templates/dashboard/product/product_type/list.html

```diff
@@ -60,14 +60,14 @@
                   </td>
                   <td>
                     {% if product_attributes %}
-                      {{ product_attributes|join:", " }}
+                      {{ product_attributes.product_attributes_sorted|join:", " }}
```

Comment:
Ideally, this should be provided by the view already in the correct order so template just renders it.